packages/utils/zarr-utils/src/normalize.ts

```diff
@@ -0,0 +1,12 @@
+// Adapted from https://github.com/hms-dbmi/vizarr/blob/5b0e3ea6fbb42d19d0e38e60e49bb73d1aca0693/src/utils.ts#L26
+import type { Readable } from '@zarrita/storage';
+import { root as zarrRoot, FetchStore } from 'zarrita';
+import { lru } from './lru-store.js';
+
+
+export async function openLru(url: string, requestInit: RequestInit) {
+  const store: Readable = new FetchStore(url, { overrides: requestInit });
+
+  // Wrap remote stores in a cache
+  return zarrRoot(lru(store));
+}
```

Comment:
I don't think this needs to be async. `zarrRoot` is just an alias for `new zarr.Location(store, "/")`

Reply:
Ah thanks for clarifying this. I think I had gotten confused at some point when I was trying to access `.attrs` on the object returned by `root()` and was trying different things until I found that I had to do `open(root())`